### PR TITLE
Process phenotypes

### PR DESCRIPTION
**Pull request by Thatguy027**
the process_phenotypes function is put together. 93.15% coverage with tests, the missing 7% is probably not necessary to include in the function. four outlier pruning methods.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -14,7 +14,8 @@ Imports:
     dplyr,
     tidyverse,
     data.table,
-    RSQLite
+    RSQLite,
+    sommer
```
Comment (danielecook) [inline DESCRIPTION]: Remove packages if you aren't using them.

File: R/visualize_outliers.R [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# df_outliers%>%
```
Comment (danielecook) [inline R/visualize_outliers.R]: File is completely commented out.

File: inst/extdata/test_phenotype.tsv [hunk 3]
```diff
@@ -0,0 +1,7470 @@
+strain	trait1	trait2	trait3	trait4
```
Comment (danielecook) [inline inst/extdata/test_phenotype.tsv]: I know we talked about this format. It's highly redundant. Not a big deal but not my fav either.

File: R/process_phenotypes.R [hunk 4]
```diff
@@ -0,0 +1,342 @@
+#' Resolve strain names to isotypes
+#'
+#' \code{resolve_isotypes} takes a vector of strain names and converts them to CeNDR-defined isotype names
+#'
+#' @param strains2resolve a vector of strain names to change to isotype names.
+#' @param isotype_lookup a tibble that contains two columns - strain, previous_name, isotype.
+#' This is used to change input strain names. Default option queries CeNDR isotypes
+#' @return Output is a vector with isotype names
+#' @export
+resolve_isotypes <- function(strains2resolve,
```
Comment (danielecook) [inline R/process_phenotypes.R]: It is rare to see intention as part of a function parameter.

File: inst/extdata/test_phenotype.tsv (hunk 3 above)
Comment (Thatguy027) [reply inst/extdata/test_phenotype.tsv]: why is this redundant?

File: R/process_phenotypes.R (hunk 4 above)
Comment (Thatguy027) [reply R/process_phenotypes.R]: whatchyu mean?

Comment (danielecook) [reply R/process_phenotypes.R]: The name `strains2resolve` is a format not often seen and describes what will happen to the variable. That's not what you usually see. It should probably be just `strains`, the function name describes what is going to happen. The current format could makes the parameter name inconsistent with other functions and therefore more confusing.

Comment (Thatguy027) [reply R/process_phenotypes.R]: ah, gotchyu

Comment (Thatguy027) [reply R/process_phenotypes.R]: done

File: DESCRIPTION (hunk 1 above)
Comment (Thatguy027) [reply DESCRIPTION]: done

**Comment by Thatguy027** [comment]
looks like i showed travis who the real boss is!

**Pull request merged**